deploy/chaos/loadtest/locustfile.py

```diff
@@ -52,11 +56,88 @@ def channel_list(self):
         """
         Load the channel page and the important endpoints.
         """
-        resp = self.client.get("/channels/")
+        self.client.get("/channels/")
         self.channel_list_api_calls()
 
+
+class ChannelPage(BaseTaskSet):
+    def on_start(self):
+        self._login()
+
+    def get_first_public_channel_id(self):
+        resp = self.client.get("/get_user_public_channels/").json()
```

Comment:
Can you add docstrings for each of these utility functions?

Reply:
Done